system.prop

```diff
@@ -1,6 +1,40 @@
-#
```

Comment:
I would like to keep thoses comments. It helps to have a separation between what is system generated and constant from the device tree.